src/components/PricePlanList.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+import PricePlan from './PricePlan';
+import { PricePlans } from '../types';
+
+interface Props {
+  plans: PricePlans;
+  selected: string;
+  onHandleClick: (val: string) => void;
+}
+
+const PricePlanList = ({ plans, selected, onHandleClick }: Props) => {
```

Comment:
Ditto: onHandleClick should be onClick

Reply:
I have updated this to also be onSelect, as the prop being passed is called selected. Would this make more sense?

Reply:
Indeed it does